tests/TestRegistryTest.cpp

```diff
@@ -326,16 +329,20 @@ TEST(TestRegistry, ResetPluginsWorks)
 
 TEST(TestRegistry, listGroupNames)
 {
+    // run tests backward, ensure that GROUP_1 will be listed
+    // after GROUP_11, and also that GROUP_2 will be listed only once
```

Comment:
Would you mind adding this text to the name of the test instead of adding it ad a comment ;)